azure-pipelines.yml

```diff
@@ -148,3 +148,33 @@ stages:
               ./test.sh --configuration $(_BuildConfig) --ci --test --integrationTest --nobl
             name: Test
             displayName: Tests
+
+      - job: MacOS
```

Comment:
I think you'll need to update RID here:

https://github.com/microsoft/testfx/blob/ff1568833d2edf25ef6c883a93a42e5261a06c8a/test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/Helpers/AcceptanceTestBase.cs#L79